### PR TITLE
13 add job expired certifications

### PR DESCRIPTION
**Pull request by decembrya**
### What does it fix?

Part of #13 

### How has it been tested?

integration testing, manual testing

Email notification sender implementation to follow in subsequent PR

### CODE REVIEW

File: Src/Tests/DeUrgenta.Tests.Helpers/Builders/UserBuilder.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        private Guid _userId = Guid.NewGuid();
+        private string _sub = Guid.NewGuid().ToString();
+
+        public User Build() => new()
+        {
+            Id = _userId,
+            FirstName = TestDataProviders.RandomString(),
+            LastName = TestDataProviders.RandomString(),
+            Sub = _sub,
+            Email = "a@a.a"
```
Comment (idormenco) [inline Src/Tests/DeUrgenta.Tests.Helpers/Builders/UserBuilder.cs]: use $"{_userId}@example.com" for email

File: Src/DeUrgenta.RecurringJobs/Jobs/ExpiredCertificationJob.cs [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        public ExpiredCertificationJob(DeUrgentaContext context, INotificationService notificationService, IOptionsMonitor<ExpiredCertificationJobConfig> config)
+        {
+            _context = context;
+            _notificationService = notificationService;
+            _config = config.CurrentValue;
+        }
+
+        public async Task RunAsync()
+        {
+            var usersWithExpiredCertifications = _context.Certifications
```
Comment (idormenco) [inline Src/DeUrgenta.RecurringJobs/Jobs/ExpiredCertificationJob.cs]: we should make sure we will not spam users.
I think we should find a mechanism to exclude previously notified users. 

We should send 2 emails:
* one x days before the certification expiration
* one when certification is expiried
A simple solution is to keep track of notified users about their certifications. Add a database context with a couple of tables but make sure they will have a "jobs" schema so they will not interfere with other tables.
Please implement a mechanism for cleaning these exclusion tables.

**Review by idormenco — CHANGES_REQUESTED** [review]

File: Src/DeUrgenta.RecurringJobs/Jobs/ExpiredCertificationJob.cs (hunk 2 above)
Comment (decembrya) [reply Src/DeUrgenta.RecurringJobs/Jobs/ExpiredCertificationJob.cs]: Done, and added #107 to add the implementation for the Email sender as discussed.

**Review by idormenco — APPROVED** [review]
lgtm

**Pull request merged**